bench/vaporetto-bench/build.rs

```diff
@@ -3,18 +3,16 @@
 use std::convert::TryFrom;
 use std::env;
 use std::fs::File;
-use std::io::{BufWriter, Cursor, Write};
+use std::io::{BufWriter, Cursor};
 use std::path::PathBuf;
 
-use vaporetto::{KyteaModel, Model, Predictor};
+use vaporetto::{KyteaModel, Model};
 
 fn main() {
-    let mut model_data = Cursor::new(include_bytes!("../../resources/jp-0.4.7-5.mod"));
-    let model = KyteaModel::read(&mut model_data).unwrap();
+    let model_data = Cursor::new(include_bytes!("../../resources/jp-0.4.7-5.mod"));
+    let model = KyteaModel::read(model_data).unwrap();
     let model = Model::try_from(model).unwrap();
-    let predictor = Predictor::new(model, false).unwrap();
     let out = &PathBuf::from(env::var_os("OUT_DIR").unwrap());
     let mut buf = BufWriter::new(File::create(out.join("predictor.bin")).unwrap());
-    let model_data = predictor.serialize_to_vec().unwrap();
```

Comment:
Could you please keep the serialization?

Reply:
Oh, sorry, I misunderstood that `Predictor::serialize_to_vec` was gone. Reverted.

https://github.com/legalforce-research/tokenizer-speed-bench/pull/1/commits/b3fe081c9786f1b24c1a844ada727a239ea4e4b2